crime-commons-spring-boot-starter-rest-client/src/test/java/uk/gov/justice/laa/crime/commons/config/RestClientAutoConfigurationTest.java

```diff
@@ -230,6 +232,16 @@ void givenRegistrationIdGetExchangeFilterWithPopulatesCorrectOAuthAttributes() {
                 });
     }
 
+    @Test
+    void traceIdHandlerIsConditionalOnTracerBean() {
+        this.contextRunner
+                .run((context) -> assertThat(context).doesNotHaveBean(TraceIdHandler.class));
+
+        this.contextRunner
+                .withUserConfiguration(TracerTestConfig.class)
```

Comment:
I think I would use the `BraveAutoConfiguration.class` here instead.

Reply:
I thought of keeping the test neutral. It can be Brave or OpenTelemetry. But it is a valid point as we are using the Brave tracing bridge.